case-studies/bayren.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: case-study
+
+client: bayren
+
+title: Bay Area Regional Energy Network (BayREN)
+
+tags: [Our Work]
+
+description: "An example of our work – an eTool - for the Bay Area Regional Energy Network."
+
+client-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
+
+challenge-text: As a component of a Multifamily Subprogram and the Codes and Standards Subprogram to provide services to the members of BayREN, the Association of Bay Area Governments (ABAG) was interested in options regarding how to implement an online or at permit counter mobile system through which permit applicants answer a set of questions to determine their energy requirements for the following permit types:<ol class="font-size-md font-family-lato line-height-2"><li>Residential Water Heating Replacement</li><li>Residential Roof Replacement</li><li>Residential Window Replacement</li></ol>
+
+
+technologies: [javascript, angular, node, docker, kubernetes, mysql]
+
+outcome-text: We designed and developed an eTool for the areas within BayREN .The homeowner uses the app to fill out details about the project and if their jurisdiction is participating in the program, they get a custom note to fill in their permit application, or get links to the correct permitting information based on their individual project. Customized a pdf download which customized the form.<br><br>Our application streamlines and clarifies the permitting process, and helps educate homeowners and contractors in choosing retrofits that match state and local regulations. Deliverables for this project included wireframes, training materials, and support for both expanded development and implementation of the eTool (including pre- and post-deployment)."
```

Comment:
adjust spacing:
within BayREN .The homeowner
should be:
within BayREN. The homeowner

this sentence needs a subject:
Customized a pdf download which customized the form.
I think 'We' should be added to the front:
We customized a pdf download which customized the form.

Reply:
Fixed typos.